docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,262 @@
+---
+id: dt-kit-interaction-patterns
+title: Interaction Patterns
+description: 'Digital Twin KIT - Interaction Patterns'
+sidebar_position: 2
+---
+
+## Interaction Patterns for Distributed Digital Twins
+
+The Catena-X standard CX-0002 (Digital Twins in Catena-X) defines a subset of the AAS-standard that is relevant for 
+Catena-X. However, it is concerned only with the network-facing APIs which in some cases may have to be augmented to 
+cover use-case requirements.
+
+While the assumptions are explicitly listed, all patterns assume that both partners involved are properly onboarded to
+the Catena-X dataspace and thus have their connectors registered with the EDC Discovery Service like so:
+
+```mermaid
```

Comment:
why are we explaining this diagram? Nothing specific to Twins in there... ==> remove

Reply:
It's not specific to twins but relevant. That's why the API is part of this kit.